src/openzeppelin/tests/security/test_initializable.cairo

```diff
@@ -0,0 +1,23 @@
+use openzeppelin::security::initializable::Initializable;
+
+fn STATE() -> Initializable::ContractState {
+    Initializable::contract_state_for_testing()
+}
+
+#[test]
+#[available_gas(2000000)]
+fn test_initialize() {
+    let mut state = STATE();
+    assert(!Initializable::StorageTrait::is_initialized(@state), 'Should not be initialized');
```

Comment:
```suggestion
use openzeppelin::security::initializable::Initializable;
use openzeppelin::security::initializable::Initializable::StorageImpl as internal;

fn STATE() -> Initializable::ContractState {
    Initializable::contract_state_for_testing()
}

#[test]
#[available_gas(2000000)]
fn test_initialize() {
    let mut state = STATE();
    assert(!internal::is_initialized(@state), 'Should not be initialized');
```

Since we are repeating Initializable::StorageImpl (the same with my PRs), a pattern like this could be more readable. I personally like the name of the trait in snake case. I used internal here because I think the trait should be called InternalTrait instead of StorageTrait.

Reply:
The name of the implementation may be enough if is not too large. For example, we may use `InternalTrait` as it is instead of `internal`, but both look better than `Initializable::InternalTrait` (too large for being repeated a lot)

Reply:
> Since we are repeating Initializable::StorageImpl (the same with my PRs), a pattern like this could be more readable. I personally like the name of the trait in snake case. I used internal here because I think the trait should be called InternalTrait instead of StorageTrait. Wdyt?

Yes, this is definitely much more readable. I want to like `internal` because it's shorter and easier to write, but I'm leaning toward `InternalTrait` because IMO it's easier for the reader to know where it's coming from

> The name of the implementation may be enough if is not too large. For example, we may use InternalTrait as it is instead of internal, but both look better than Initializable::InternalTrait (too large for being repeated a lot)

Agreed, I vote for `InternalTrait` as I don't think it's too large